test/multipart.test.js

```diff
@@ -136,7 +136,7 @@ test('should group parts with the same name to an array', function (t) {
   fastify.register(multipart)
 
   fastify.post('/', async function (req, reply) {
-    const parts = await req.multipartIterator()
```

Comment:
If we keep `multipartIterator` at least one test should ensure the existence.

Reply:
done